app/main.py

```diff
@@ -1,3 +1,22 @@
+import math
+
+
 class OnlineCourse:
-    # write your code here
-    pass
+
+    weeks = 0
+
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        return math.ceil(days / 7)
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        cls.weeks = OnlineCourse.days_to_weeks(course_dict["days"])
+        course = OnlineCourse(
```

Comment:
It's better to use `cls` to access the class in classmethods.
The reasons will be discussed in the future topics on OOP :)

Reply:
i'm changed to **self** but pyCharm show me it litle bit incorrect(without bugs), and also in example video everywhere uses **cls** in classmethod method

Reply:
Do not use self in classmethods :) 
```suggestion
        course = cls(
```
Just use cls instead of using the class name `OnlineCourse`

Reply:
thanks) changed)